Sources/Views/ALKInformationCell.swift

```diff
@@ -91,10 +91,11 @@ final class ALKInformationCell: UITableViewCell, Localizable {
                                                                    options: NSStringDrawingOptions.usesLineFragmentOrigin,
                                                                    attributes: [NSAttributedString.Key.font: ALKMessageStyle.infoMessage.font],
                                                                    context: nil)
-            //  Get feedback dictionary for view 
-            guard let dictionary = ALKInformationCell().getFeedback(viewModel: viewModel) else { return 0 }
-            if dictionary["comments"] != nil {
+            //  Get feedback dictionary for view
+            if let dictionary = ALKInformationCell().getFeedback(viewModel: viewModel),dictionary["comments"] != nil  {
                 messageHeigh = (rect.height + Padding.MessageView.height + Padding.CommentView.height)
+            } else if let message = viewModel.message , !message.isEmpty {
+                messageHeigh = (rect.height + Padding.MessageView.height)
             } else {
                 messageHeigh = rect.height + Padding.MessageView.height
```

Comment:
We can merge both the `else` conditions, they are doing the same thing.

Reply:
done